.vscode/launch.json

```diff
@@ -0,0 +1,46 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Boomer",
+      "type": "lldb",
+      "request": "launch",
+      "program": "${workspaceFolder}/target/debug/boomer",
+      "args": ["-vvv", "--device", "/dev/dri/card${input:dri-device-number}"],
+      "cwd": "${workspaceFolder}",
+      "env": {
+        "RUST_BACKTRACE": "full"
+      },
+      "preLaunchTask": "build-debug"
+    },
+    {
+      "name": "Dradis (test-single-mode)",
+      "type": "lldb",
+      "request": "launch",
+      "program": "${workspaceFolder}/target/debug/dradis",
+      "args": ["-vvv", "--device", "/dev/media${input:media-device-number}", "--dump-frames", "corrupted", "--dump-frames-limit", "3", "${workspaceFolder}/dradis/samples/test-single-mode-720p.yaml"],
+      "cwd": "${workspaceFolder}",
+      "env": {
+        "RUST_BACKTRACE": "full"
+      },
+      "preLaunchTask": "build-debug"
+    }
+  ],
+  "inputs": [
+    {
+      "id": "dri-device-number",
+      "type": "promptString",
+      "default": "0",
+      "description": "Device number, by default 0 (/dev/dri/card0)."
+    },
+    {
+      "id": "media-device-number",
+      "type": "promptString",
+      "default": "0",
+      "description": "Device number, by default 0 (/dev/media0). Run `v4l2-ctl --list-devices` to find the correct number."
+    }
+  ]
+}
```

Comment:
So afaiu, this is to run vscode directly on the RaspberryPi5? Do we really expect that to be the main development setup? How does it work if the development happens on a separate machine?